src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -196,6 +206,27 @@ public void PrepareToInstallPackages()
             this.NuGetBindingOperation.PrepareOnUIThread();
         }
 
+        public async Task<bool> SaveServerExclusionsAsync(CancellationToken cancellationToken)
+        {
+            try
+            {
+                var configurationProviderService = host.GetService<IConfigurationProviderService>();
+                var sonarQubeService = host.SonarQubeService;
+                var projectKey = configurationProviderService.GetConfiguration().Project.ProjectKey;
+
+                var exclusions = await sonarQubeService.GetServerExclusions(projectKey, cancellationToken);
+
+                exclusionSettingsStorage.SaveSettings(exclusions);
+            }
+            catch (Exception ex)
+            {
+                host.Logger.WriteLine(string.Format(Strings.SaveExclusionsFailed, ex.Message));
```

Comment:
The message being logged is
> Failed to save exclusion file to the disk. Server Side exclusions may fail. Error: {0}.

... and it will be logged if the webservice call failed, which would be confusing.

Doesn't the exclusion settings class already log an exception detailing why the save failed? Either way, this message should perhaps be more generic e.g. "Failed to retrieve/process server settings. Some analysis settings will not be applied (e.g. file inclusions/exclusion)."

Reply:
If we remove the catch this comment should be removed as well there are no scenarios in our code that will fail without an exception.

Reply:
I'm not suggesting removing this catch; it's at the outer level, and there are other things that could fail e.g. the webservice call.

I'm asking whether the output message we are logging overall will make sense to the user.

Reply:
As I understand you suggested removing the catch in the previous comment.

Reply:
That wasn't what I was suggesting.